app/src/main/java/com/islamey/hamza/wazaker/domain/DataSets/MorningAzkarDataset.kt

```diff
@@ -1,6 +1,6 @@
-package com.islamey.hamza.wazaker.data.DataSets
+package com.islamey.hamza.wazaker.domain.DataSets
 
-import com.islamey.hamza.wazaker.data.Models.ZekerModel
+import com.islamey.hamza.wazaker.domain.Models.ZekerModel
```

Comment:
why moved to domain layer ?

Reply:
as it's a new arch for the app, the models should be in the domain layer ..
Data layer contains the remote (api) or local data and the domain of the app contains the models used in the app